rocketpool-cli/wallet/status.go

```diff
@@ -33,13 +34,27 @@ func getStatus(c *cli.Context) error {
 		return err
 	}
 
-	// Print status & return
-	if status.WalletInitialized {
-		fmt.Println("The node wallet is initialized.")
-		fmt.Printf("Node account: %s\n", status.AccountAddress.Hex())
-	} else {
-		fmt.Println("The node wallet has not been initialized.")
+	// Masquerading
+	emptyAddress := common.Address{}
+	if status.IsMasquerading && status.NodeAddress != emptyAddress {
+		fmt.Printf("The node wallet is initialized, but you are currently masquerading as %s%s%s\n", colorBlue, status.AccountAddress, colorReset)
+		fmt.Printf("Wallet Address: %s\n", status.NodeAddress)
+		fmt.Printf("%sDue to this mismatch, the node cannot submit transactions. Use the command 'rocketpool wallet end-masquerade' to end masquerading and restore your wallet address.%s", colorYellow, colorReset)
+	} else if status.IsMasquerading {
+		fmt.Printf("The node wallet has not been initialized, but you are currently masquerading as %s%s%s\n", colorBlue, status.AccountAddress, colorReset)
+		fmt.Printf("%sThe node cannot submit transactions. Use the command 'rocketpool wallet end-masquerade' to end masquerading.%s", colorYellow, colorReset)
```

Comment:
```suggestion
	if status.IsMasquerading {
		if status.NodeAddress != emptyAddress {
			fmt.Printf("The node wallet is initialized, but you are currently masquerading as %s%s%s\n", colorBlue, status.AccountAddress, colorReset)
			fmt.Printf("Wallet Address: %s\n", status.NodeAddress)
			fmt.Printf("%sDue to this mismatch, the node cannot submit transactions. Use the command 'rocketpool wallet end-masquerade' to end masquerading and restore your wallet address.%s", colorYellow, colorReset)
		} else {
			fmt.Printf("The node wallet has not been initialized, but you are currently masquerading as %s%s%s\n", colorBlue, status.AccountAddress, colorReset)
			fmt.Printf("%sThe node cannot submit transactions. Use the command 'rocketpool wallet end-masquerade' to end masquerading.%s", colorYellow, colorReset)
		}
```

Normally I'm a fan of being as outdented as possible, but not if it requires duplicate conditionals between an `if` and `else if`